.github/workflows/integration.yml

```diff
@@ -235,9 +179,96 @@ jobs:
       - uses: actions/upload-artifact@v3
         if: always()
         with:
-          name: integration_results_${{ matrix.python-version }}_${{ matrix.os }}_${{ matrix.adapter }}-${{ steps.date.outputs.date }}.csv
+          name: integration_results_${{ matrix.python-version }}_${{ matrix.os }}_redshift-${{ steps.date.outputs.date }}.csv
           path: integration_results.csv
 
+  test-flaky:
```

Comment:
I skipped some steps from the non-flaky (original) version:
- artifacts are not uploaded
- AWS profile is not configured (none of the tests that would use this have been identified as flaky)